lib/ClimaLandSimulations/src/Fluxnet/fluxnet_sites/US-Ha1/US-Ha1_simulation.jl

```diff
@@ -0,0 +1,28 @@
+export harvard_default_args
+
+"""
+    harvard_default_args = (
+        dz_bottom = FT(1.5),
+        dz_top = FT(0.025),
+        n_stem = Int64(1),
+        n_leaf = Int64(1),
+        h_leaf = FT(12), # m
+        h_stem = FT(14), # m
+        t0 = Float64(120 * 3600 * 24),
+        dt = Float64(40),
+        n = 45,
```

Comment:
is it ever mentioned what `n` is? or what these parameters mean?

Reply:
good point. I am not sure. (this is copied from experiments). 

More broadly, in future PRs, we definitely need to make better docstring throughout